secator/loader.py

```
@@ -28,6 +28,32 @@ def _file_has_exporter(path):
 		return False
 
 
+def _warn_duplicate_names(entries, _seen=set()):
```

Ruff (0.16.1)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)